galley/pkg/runtime/conversions/auth_test.go

```diff
@@ -0,0 +1,125 @@
+//  Copyright 2019 Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package conversions
+
+import (
+	"istio.io/istio/galley/pkg/runtime/resource"
+	"reflect"
+	"sort"
+	"testing"
+)
+
+func TestAuthConverter(t *testing.T) {
+	ac := AuthConverter{}
+
+	services := []struct {
+		Name      string
+		Namespace string
+		Selector  resource.Labels
+	}{
+		{Name: "svc1", Namespace: "ns1", Selector: resource.Labels{"a": "1", "b": "1"}},
```

Comment:
May be remove `b` labels if it doesn't contribute to the test cases (in other words, adding this only for service that using the same label `a`).

Also, may be use `app` and `version` so it's easy to make a mental map to the real world setup.

Reply:
Changed to user `version` only.